node/core/provisioner/src/disputes/prioritized_selection/mod.rs

```diff
@@ -41,7 +41,7 @@ mod tests;
 /// The maximum number of disputes Provisioner will include in the inherent data.
 /// Serves as a protection not to flood the Runtime with excessive data.
 #[cfg(not(test))]
-pub const MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME: usize = 200_000;
+pub const MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME: usize = 1_000;
```

Comment:
If this is something that is controlled by the node side, this is a security issue. The runtime should ensure that it doesn't accept too many votes.

Reply:
The actual number of disputes included in a block is limited by the runtime. This limit controls just how much data we send it with the inherent data.